### DISCUSSION TITLE
Admin app navigation "open in new tab" gone

### DISCUSSION THREAD

**Discussion by aggelito** [directus/directus#13835] [Feature Requests]
Just noticed you can't open new tabs by right click -> open in new tab or by clicking the scroll wheel on a link anymore.

I'm used to having multiple tabs open so it's easier to navigate between pages I'm frequently using.

I think it would be better UX to have this feature unless theres a really good reason for it of course.

Or am i the only one that's not a big fan of these "javascript links"?

**Comment by br41nslug** [comment]
We're looking into it. This was not an intentional change but most likely a regression in one of our dependencies.

**Comment by bcartier** [comment]
It would be great to allow default browser behaviour for links. In the meantime @aggelito ,  ctrl+click (or command+click on a mac) still seems to work throughout the admin app.

**Reply by sxsws** [reply]
ctrl+click doesn't in latest version.

**Reply by benhaynes** [reply]
I think that is due to a dependency change within _vue-router_... it's being tracked in another ticket and will be resolved.

**Reply by rijkvanzanten** [reply]
https://github.com/directus/directus/pull/13840 🎉

**Comment by nickrum** [comment]
This has been addressed in #13840
